src/js/out_queue.js

```diff
@@ -51,20 +51,28 @@
 	 * @param object mutSnowplowState Gives the pageUnloadGuard a reference to the outbound queue
 	 *                                so it can unload the page when all queues are empty
 	 * @param boolean useLocalStorage Whether to use localStorage at all
-	 * @param boolean usePost Whether to send events by POST or GET
+	 * @param string eventMethod if null will use 'beacon' otherwise can be set to 'post', 'get', or 'beacon' to force. 
 	 * @param int bufferSize How many events to batch in localStorage before sending them all.
 	 *                       Only applies when sending POST requests and when localStorage is available.
 	 * @param int maxPostBytes Maximum combined size in bytes of the event JSONs in a POST request
 	 * @return object OutQueueManager instance
 	 */
-	object.OutQueueManager = function (functionName, namespace, mutSnowplowState, useLocalStorage, useBeacon, usePost, bufferSize, maxPostBytes) {
+	object.OutQueueManager = function (functionName, namespace, mutSnowplowState, useLocalStorage, eventMethod, bufferSize, maxPostBytes) {
 		var	queueName,
 			executingQueue = false,
 			configCollectorUrl,
 			outQueue;
-
-		useBeacon = useBeacon && navigator && navigator.sendBeacon;
-
+		
+		//Force to lower case if its a string
+		eventMethod = eventMethod.toLowerCase ? eventMethod.toLowerCase() : eventMethod;
+		
+		//Use the Beacon API if eventMethod is set null, true, or 'beacon'. 
+		var enableBeacon = (eventMethod === null || eventMethod === true || eventMethod === "beacon" || eventMethod === "true") ? true : false;
+		// Fall back to POST or GET for browsers which don't support Beacon API
+		useBeacon = enableBeacon && navigator && navigator.sendBeacon;
```

Comment:
this needs to be declared (I fixed it before deploying, but just pointing it out)